README.md

```diff
@@ -16,3 +16,53 @@ The available endpoints are:
 - `/blogs/new`
 - `/posts/<string:blog_id>`
 - `/posts/new/<string:blog_id>`
+
+
+## HOW TO USE THIS CODE
+
+1) Create a new virtual environment: `python -m venv venv`
```

Comment:
To stick to what we do in other parts of the course (especially the new parts), could you use `.venv` as the venv name?

Reply:
Updated